uber/site_sections/reg_admin.py

```diff
@@ -15,12 +15,17 @@
 @all_renderable()
 class Root:
     def import_attendees(self, session, target_server='', api_token='', query='', message=''):
+        load_message = message
         service, message, target_url = get_api_service_from_server(target_server, api_token)
+        message = message or load_message
```

Comment:
I suspect that this pattern only confused me because I got home after 11pm tonight, so you should definitely take this with a huge grain of salt, but I had to read through this a few times to see why we needed two message variables.

I suggest eliminating ``load_message`` and for the second message variable doing this:

```python
service, service_message, target_url = get_api_service_from_server(target_server, api_token)
message = service_message or message
```

which is 1 line of code shorter and is either less confusing or I'm just tired :P

Reply:
Oh yea, that's a much better way to do it!